src/components/Content/ParticipantList/Participant/Participant.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const Participant = ({ username }) => {
+  return <div className="participant-ctn">
+    <div className="participant-avatar"></div>
+    <div className="participant-name">
+      {username}
```

Comment:
@Moe-Shoman Let's make a User widget, or Member widget, and use it all over the place -- when we show users' usernames in ^^this^^, when users are mentioned in LeapChat (and we render the markdown to HTML that includes this widget), when a user is mentioned in the description/deliverables of a task, etc.